install/mysql/glpi-empty.sql

```diff
@@ -7590,7 +7590,7 @@ CREATE TABLE `glpi_users` (
   `date_sync` timestamp NULL DEFAULT NULL,
   `is_deleted` tinyint NOT NULL DEFAULT '0',
   `profiles_id` int unsigned NOT NULL DEFAULT '0',
-  `entities_id` int unsigned NOT NULL DEFAULT '0',
+  `entities_id` int unsigned DEFAULT '0',
```

Comment:
Maybe the default value should be `null`. It would probably simplify the default value when a user is created. Also, there is a `if (!isset($input["entities_id"])) { $input["entities_id"] = 0; }` in the `User::prepareInputForAdd()` that should probably be updated or removed.

```suggestion
  `entities_id` int unsigned DEFAULT NULL,
```